datasets/indonlu/indonlu.py

```diff
@@ -0,0 +1,631 @@
+                text_feature: datasets.Value("string") for text_feature in six.iterkeys(self.config.text_features)
+            }
+
+        if self.config.label_classes:
+            if self.config.name in sentence_features:
+                features["seq_label"] = datasets.Sequence(
+                    datasets.features.ClassLabel(names=self.config.label_classes)
+                )
+            elif self.config.name in ner_:
+                features["ner_tags"] = datasets.Sequence(datasets.features.ClassLabel(names=self.config.label_classes))
+            elif self.config.name in pos_:
+                features["pos_tags"] = datasets.Sequence(datasets.features.ClassLabel(names=self.config.label_classes))
+            elif self.config.name == "casa" or self.config.name == "hoasa":
+                for label in self.config.label_column:
+                    features[label] = datasets.features.ClassLabel(names=self.config.label_classes)
+            else:
+                features["label"] = datasets.features.ClassLabel(names=self.config.label_classes)
+
+        return datasets.DatasetInfo(
+            description=self.config.description,
+            features=datasets.Features(features),
+            homepage=_INDONLU_HOMEPAGE,
+            citation=self.config.citation + "\n" + _INDONLU_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
+        """Returns SplitGenerators."""
+        train_path = dl_manager.download_and_extract(self.config.train_url)
+        valid_path = dl_manager.download_and_extract(self.config.valid_url)
+        test_path = dl_manager.download_and_extract(self.config.test_url)
+        return [
+            datasets.SplitGenerator(name=datasets.Split.TRAIN, gen_kwargs={"filepath": train_path}),
+            datasets.SplitGenerator(name=datasets.Split.VALIDATION, gen_kwargs={"filepath": valid_path}),
+            datasets.SplitGenerator(name=datasets.Split.TEST, gen_kwargs={"filepath": test_path}),
+        ]
+
+    def _generate_examples(self, filepath):
+        """ Yields examples. """
+        csv_file = ["emot", "wrete", "facqa", "casa", "hoasa"]
+        tsv_file = ["smsa"]
+        txt_file = ["terma", "keps"]
+        txt_file_pos = ["posp", "bapos"]
+        txt_file_ner = ["nergrit", "nerp"]
+
+        with open(filepath, encoding="utf-8") as f:
+
+            if self.config.name in csv_file:
+                reader = csv.reader(f, delimiter=",", quotechar='"', quoting=csv.QUOTE_ALL)
+                next(reader)  # skip first row which is header
+
+                for id_, row in enumerate(reader):
+                    if self.config.name == "emot":
+                        label, tweet = row
+                        yield id_, {"tweet": tweet, "label": label}
+                    elif self.config.name == "wrete":
+                        sent_A, sent_B, category, label = row
+                        yield id_, {"sent_A": sent_A, "sent_B": sent_B, "category": category, "label": label}
```

Comment:
Usually for NLI datasets we use the column names `premise` and `hypothesis` instead of `sent_A` and `sent_B`.
Can you do this change please ?